elyra/templates/airflow/airflow_template.jinja2

```diff
@@ -23,9 +23,8 @@ dag = DAG(
 {% endfor %}
 {% else %}
 from airflow.kubernetes.secret import Secret
```

Comment:
@lresende @MR-GOYAL @kevin-bates @ianonavy interesting ... from Airflow 2.7.0 on, the secrets line changed to

The one notable exception is that we will **continue to support the airflow.providers.cncf.kubernetes.secret.Secret class**.

at https://airflow.apache.org/docs/apache-airflow/2.7.0/howto/upgrading-from-1-10/index.html#step-5-upgrade-airflow-dags

vs. before 2.7.0

The one notable exception is that **we will continue to support the airflow.kubernetes.secret.Secret class**.

at https://airflow.apache.org/docs/apache-airflow/2.6.2/howto/upgrading-from-1-10/index.html#step-5-upgrade-airflow-dags

Now, with Airflow 2.8.2, `from airflow.kubernetes.secret import Secret` is still working fine, but I should probably change it soon to `from airflow.providers.cncf.kubernetes.secret import Secret`